scripts/gha/setup-training-operator.sh

```diff
@@ -42,6 +42,16 @@ if [ "${GANG_SCHEDULER_NAME}" = "scheduler-plugins" ]; then
   echo "Configure gang-scheduling using scheduler-plugins to training-operator"
   kubectl patch -n kubeflow deployments training-operator --type='json' \
     -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/1", "value": "--gang-scheduler-name=scheduler-plugins"}]'
+elif  [ "${GANG_SCHEDULER_NAME}" = "volcano" ]; then
+  VOLCANO_SCHEDULER_VERSION=$(go list -m -f "{{.Version}}" volcano.sh/apis|grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+')
```

Comment:
Why do we need to do `grep`?

Reply:
Can we have a cleaner solution?

Reply:
I think we can get a version to do `go list -m -f "{{.Version}}" volcano.sh/apis` without `grep`.
Then, a link is including the version: https://raw.githubusercontent.com/volcano-sh/volcano/v1.7.0/installer/volcano-development.yaml

```shell
$ go list -m -f "{{.Version}}" volcano.sh/apis
v1.7.0
```

Reply:
At first, I also used `go list -m -f "{{.Version}}" volcano.sh/apis`. But there are other apis release as `v1.2.0-k8s1.19.6` which has no relative release of volcano. So I change it to this one for more compatible.

If you feel `go list -m -f "{{.Version}}" volcano.sh/apis` is ok, I will change it back.

Reply:
@lowang-bh In recent releases, volcano used semantic versioning as the release name.
So, `go list -m -f "{{.Version}}" volcano.sh/apis` is enough. 

https://github.com/volcano-sh/apis/tags

Reply:
Done.